### PR TITLE
feat: unify requirements in pyproject

### PR DESCRIPTION
**Pull request by vncsna**
WDYT sobre unificar as dependências no pyproject.toml?

### CODE REVIEW

File: .github/workflows/cd-staging.yaml [hunk 1]
```diff
@@ -39,21 +43,18 @@ jobs:
           export_default_credentials: true
       - name: Install Python dependencies for deploying
         run: |-
-          pip install -r requirements-deploy.txt
+          poetry install with=dev
```
Comment (aspeddro) [inline .github/workflows/cd-staging.yaml:46]:
```suggestion
          poetry install --with=dev
```

Testei aqui https://github.com/basedosdados/pipelines/pull/660

Log https://github.com/basedosdados/pipelines/actions/runs/7920747397/job/21624461290?pr=660#step:6:27

File: .github/workflows/cd.yaml [hunk 2]
```diff
@@ -38,21 +42,18 @@ jobs:
           export_default_credentials: true
       - name: Install Python dependencies for deploying
         run: |-
-          pip install -r requirements-deploy.txt
+          poetry install with=dev
```
Comment (aspeddro) [inline .github/workflows/cd.yaml:45]: flag aqui

File: .github/workflows/cd-staging.yaml (hunk 1 above)
Comment (vncsna) [reply .github/workflows/cd-staging.yaml]: Valeu, corrigi abaixo

**Review by aspeddro — APPROVED** [review]
🎉

**Pull request merged**